internal/pkg/daemon/seccompprofile/seccompprofile.go

```diff
@@ -88,7 +87,6 @@ type Reconciler struct {
 	record  event.Recorder
 	save    saver
 	metrics *metrics.Metrics
-	ready   atomic.Bool
```

Comment:
The idea of this indicator was to become ready once one successful reconciliation loop has been done. We indeed install a bunch of default profiles via the manager, which then should get reconciled by the daemon set: https://github.com/kubernetes-sigs/security-profiles-operator/blob/664598434c3c896d6a69a35f8313a07abdcdf482/internal/pkg/manager/spod/bindata/default_profiles.go#L28

Reply:
I saw this profile being sometimes installed. Somehow it doesn't get always installed which leads to the failure mentioned above. 

I would rather not have this profile installed by default because we don't have any nginx running in the cluster.